pkg/api/unversioned/group_version.go

```diff
@@ -62,6 +71,11 @@ func (gv GroupVersion) String() string {
 // ParseGroupVersion turns "group/version" string into a GroupVersion struct. It reports error
 // if it cannot parse the string.
 func ParseGroupVersion(gv string) (GroupVersion, error) {
+	// this can be the internal version for kube
+	if (len(gv) == 0) || (gv == "/") {
```

Comment:
The comment is wrong, it not for kube, it's for v1.

So both "" and "/" will be interpreted as the internal version for v1? How about we only allow ""?